### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Tchilo**
Modify .css and .html for desktop version

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Tchilo, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -316,3 +320,126 @@ label {
 .footer-email {
   margin-bottom: 17.04px;
 }
+
+@media screen and (min-width: 992px) {
```
Comment (mke2111) [inline style.css:324]: > You have done a splendid job so far. 👍🏽 

- [x] Between 992px and around 1400px, you have horizontal scroll. This simply indicates that some containers are bigger than others. 
- Consider the image below, the contact container is bigger than  all the other containers. Give this a fix.
Screenshot from 2021-08-18 17-09-39
#
Screenshot from 2021-08-18 17-11-54

Comment (Tchilo) [reply style.css:324]: I have made the changes

File: index.html [hunk 2]
```diff
@@ -13,19 +13,25 @@
 
 <body>
   <header>
-    <nav class="header">
-      <a href="#">
+    <nav class="header ">
```
Comment (SajjadAhmad14) [inline index.html:16]: [ OPTIONAL ] The navigation bar is overflowing in the mobile viewport.
navigation

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi TC Chilongoshi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏
### OPTIONAL
- [ ] Navigation bar on small mobile view is overlapping the intro section. Please see the inline comments below and screenshot. 
## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**